src/util/util.ts

```diff
@@ -52,10 +52,10 @@ export function forEachReverse<T, U>(arr: readonly T[] | null | undefined, actio
 }
 
 export function earliestDate(...dates: (Date | undefined)[]) {
-    return dates.reduce((d1,d2) => (d1 && d2 && d2 < d1 ? d2 : d1) || d2, undefined);
+    return dates.reduce((d1,d2) => (d1 && d2 && d2 < d1 ? d2 : d1) || d1 || d2, undefined);
```

Comment:
Just curious about these, why the change?

Reply:
Bah.  It wasn't supposed to be there.

(I debated for a while whether `(...) || d2` is being too much of a smartass and thought that maybe it's better to add `d1` just for the sake of clarity.  Then I decided against it, but forgot to remove it.  Now I think that the `d1`/`d2` names are confusing since they're not symmetrical, so I'll do that instead.)